extensions/blocks/podcast-player/style.scss

```diff
@@ -105,30 +106,38 @@ $player-background: transparent;
 		height: $cover-image-size;
 	}
 
-	.jetpack-podcast-player__title {
-		display: flex;
-		flex-direction: column;
-		margin: 0;
-		overflow: hidden;
-	}
-
-	.jetpack-podcast-player__current-track-title {
-		font-size: $track-title-font-size;
-		margin: 0 0 $track-title-b-margin;
-	}
-
-	.jetpack-podcast-player__podcast-title {
-		font-size: $podcast-title-font-size;
-		color: $text-color;
-		margin: 0;
-	}
-
-	a.jetpack-podcast-player__podcast-title {
-		text-decoration: none;
-
-		&:hover,
-		&:focus {
-			color: $text-color-hover;
+	// Increasing specificity on the header titles and links for more consistency across themes and editor canvas.
```

Comment:
This comment looks like it could use some wrapping (with `/* ... */`). If you're using VSCode, I recommend the [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) extension 💪

Reply:
Thanks for the tip!